pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -277,6 +278,14 @@ def reset(self, metrics: Optional[bool] = None) -> None:
         self._split_idx = None
         self._current_fx = None
 
+    @property
+    def spawn_extra_parameters(self) -> Dict[str, Dict[str, _METRIC]]:
+        return self._spawn_extra_parameters
+
+    @spawn_extra_parameters.setter
+    def spawn_extra_parameters(self, val: Dict[str, Dict[str, _METRIC]]) -> None:
+        self._spawn_extra_parameters = val
```

Comment:
Since these are not customizing anything, you can have the property directly in the `__init__`